src/main/java/com/bhoruka/bloodbank/service/CampService.java

```diff
@@ -24,9 +30,15 @@ public CampService(@NonNull final CampDao campDao) {
     /**
      * Method to create a new camp.
      *
+     * @param createCampRequest the create request received from the controller
      * @return camp id
      */
-    public String createCamp() {
-        return "campId";
+    public String createCamp(@NonNull final CreateCampRequest createCampRequest) {
+        CampModel createdCamp = campDao.createCamp(CampModel.builder()
+                    .partnerId(createCampRequest.getPartnerId())
+                    .expectedNoOfDonor(createCampRequest.getExpectedNoOfDonor())
+                    .dateOfCamp(createCampRequest.getDateOfCamp())
+                    .build());
+        return createdCamp.getId();
```

Comment:
Is it wise to return null or would throwing a custom exception when ID is null make more sense?

Reply:
I think we can do that in the controller.